pkg/kfapp/aws/aws.go

```diff
@@ -467,9 +467,10 @@ func (aws *Aws) Generate(resources kftypes.ResourceEnum) error {
 		}
 	} else {
 		if pluginSpec.Auth.Cognito != nil {
-			if err := aws.kfDef.SetApplicationParameter("istio", "clusterRbacConfig", "ON"); err != nil {
-				return errors.WithStack(err)
-			}
+			// Will roll back after istio move to v3 done
```

Comment:
Do we need to roll back later?

Reply:
We don't have to, it depends on if we want to set `clusterRbacConfig=ON` in kfctl logic or manifests configmap.